accounts/keystore/passphrase.go

```diff
@@ -117,7 +117,7 @@ func (ks keyStorePassphrase) StoreKey(filename string, key *Key, auth string) er
 		// Verify that we can decrypt the file with the given password.
 		_, err = ks.GetKey(key.Address, tmpName, auth)
 		if err != nil {
-			msg := "An error was encountered when saving and verifying the keystore file. \n" +
+			msg := "an error was encountered when saving and verifying the keystore file. \n" +
```

Comment:
This is a use message as far as I see it, that's why it was capitalized. Multiple sentences.